### PR TITLE
beancount-language-server 1.3.5

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by p-linnane** [comment]
Not the latest release: https://github.com/polarmutex/beancount-language-server/releases

**Comment by chenrui333** [comment]
should be good now

image

**Comment by chenrui333** [comment]
sha stays the same.

File: .github/autobump.txt [hunk 1]
```diff
@@ -257,6 +257,7 @@ buildozer
 buildpulse-test-reporter
 buku
 bulk_extractor
+bump-beancount-language-server
```
Comment (alebcay) [inline .github/autobump.txt:260]:
```suggestion
beancount-language-server
```

**Review by carlocab — APPROVED** [review]

**Pull request merged**